SDPi_Supplement/asciidoc/plantuml/vol2-figure-dev-25-sequence.puml

```diff
@@ -0,0 +1,27 @@
+@startuml
+skinparam monochrome true
+autonumber
+
+!global $str_sdpi_p_somds_cons = "SOMDS Consumer"
+!global $str_sdpi_p_somds_prov = "SOMDS Provider"
+
+participant "$str_sdpi_p_somds_cons" as sdpi_somds_consumer
+participant "$str_sdpi_p_somds_prov" as sdpi_somds_provider
+
+== SDPi [DEV-25]  Discover BICEPS Services ==
+
+sdpi_somds_consumer -> sdpi_somds_provider: TransferGet()

```

Comment:
Let's name this GetMetadata and GetMetadataResponse for the abstract transaction